Examples/Physics_applications/laser_acceleration/inputs_1d

```diff
@@ -0,0 +1,66 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 1000
```

Comment:
Can you say how long the test runs with 2 MPI ranks  & 1 OMP thread? Just checking it's short enough for CI.

(Python test below uses 10 steps)

Reply:
Total time = 0.4985202 with 2 MPI, 1 OMP, 1000 steps

Reply:
Perfect, that's totally fine.

Do you like to increase the number of steps in the Python test, too?

Reply:
Sure, how much would you recommend?